openpype/modules/ftrack/plugins/publish/collect_ftrack_api.py

```diff
@@ -44,19 +44,25 @@ def process(self, context):
 
         self.log.debug("Project found: {0}".format(project_entity))
 
+        task_object_type = session.query(
+            "ObjectType where name is 'Task'").one()
+        task_object_type_id = task_object_type["id"]
```

Comment:
Would this query be more optimal if we were to do something like `select id from ObjectType` in the query?

Reply:
That's default behavior.